include/boost/geometry/strategies/cartesian/intersection.hpp

```diff
@@ -459,6 +459,7 @@ struct cartesian_segments
 
         bool collinear = sides.collinear();
 
+        //TODO: remove this
```

Comment:
In this PR?

Reply:
ideally in the PR that removes rescaling (I added a note)